gdx/src/com/badlogic/gdx/graphics/glutils/ETC1.java

```diff
@@ -42,7 +43,7 @@
 
 	/** Class for storing ETC1 compressed image data.
 	 * @author mzechner */
-	public final static class ETC1Data implements Disposable {
+	public static class ETC1Data implements Disposable {

```

Comment:
I need one of the following options:
- leave the **final** in place but add reading and writing from stream.
- or just remove the **final** so I can extend the class and add all the changes of this commit outside of the gdx